src/agoradatatools/great_expectations/gx/expectations/genes_biodomains.json

```diff
@@ -56,51 +56,75 @@
-      "kwargs": {
-        "column": "gene_biodomains",
-        "member_type": "dict"
-      },
-      "meta": {}
-    },
-    {
-      "expectation_type": "expect_column_values_to_have_list_of_dict_with_expected_values",
+      "expectation_type": "expect_column_values_to_match_json_schema",
       "kwargs": {
         "column": "gene_biodomains",
-        "list_dict_values": {
-          "key": "biodomain",
-          "values": [
-            "Apoptosis",
-            "Vasculature",
-            "Lipid Metabolism",
-            "Proteostasis",
-            "Immune Response",
-            "Autophagy",
-            "Mitochondrial Metabolism",
-            "Structural Stabilization",
-            "Synapse",
-            "Endolysosome",
-            "Metal Binding and Homeostasis",
-            "Oxidative Stress",
-            "Epigenetic",
-            "APP Metabolism",
-            "Cell Cycle",
-            "DNA Repair",
-            "RNA Spliceosome",
-            "Tau Homeostasis",
-            "Myelination"
-          ]
+        "json_schema": {
+          "$id": "https://github.com/Sage-Bionetworks/agora-data-tools/src/agoradatatools/great_expectations/gx/json_schemas/genes_biodomains/gene_biodomains_schema.json",
+          "$schema": "https://json-schema.org/draft/2019-09/schema",
+          "default": [],
+          "items": {
+            "default": {},
+            "properties": {
+              "biodomain": {
+                "default": "",
+                "enum": [
+                  "Apoptosis",
+                  "Vasculature",
+                  "Lipid Metabolism",
+                  "Proteostasis",
+                  "Immune Response",
+                  "Autophagy",
+                  "Mitochondrial Metabolism",
+                  "Structural Stabilization",
+                  "Synapse",
+                  "Endolysosome",
+                  "Metal Binding and Homeostasis",
+                  "Oxidative Stress",
+                  "Epigenetic",
+                  "APP Metabolism",
+                  "Cell Cycle",
+                  "DNA Repair",
+                  "RNA Spliceosome",
+                  "Tau Homeostasis",
+                  "Myelination"
+                ],
+                "type": "string"
+              },
+              "go_terms": {
+                "default": [],
+                "items": {
+                  "UniqueItems": true,
+                  "maxItems": 2,
+                  "minItems": 1,
+                  "type": "string"
+                },
+                "type": "array"
+              },
+              "n_biodomain_terms": {
+                "default": 0,
+                "type": "integer"
+              },
+              "n_gene_biodomain_terms": {
+                "default": 0,
+                "type": "integer"
+              },
+              "pct_linking_terms": {
+                "default": 0.0,
+                "type": "number"
```

Comment:
Any way to specify that this number should be between 0 and 100 inclusive?

Reply:
I'll look into it!

Reply:
I made this change in my latest commit.